docs/getting-started/zowe_faq.md

```diff
@@ -181,6 +185,17 @@ For information and tutorials about extending Zowe with a new plug-in or applica
 </details>
 
 
+### Where can I submit an idea for a future enhancement to Zowe?
+
+<details className="zowe-faq">
+
+<summary></summary>
+
+Go to the [Zowe Community ReadMe file](https://github.com/zowe/community#submit-an-issue) for information on requesting a bug fix or enhancement. Members of the Zowe community can then review your issue to post feedback or vote their support. Issues are continuously monitored by Zowe squads for improvement ideas.
```

Comment:
This may feel like an extra step for a few people, but I think the FAQ is also about teaching people where they can find this kind of info 😋 

The other alternative is to provide the direct link to open the issue. Thus changing the first sentence of this paragraph